### PR TITLE
Escapee stuff

### PR DESCRIPTION
**Pull request by Kyle2143**
## About the PR 
Adds some stats to be displayed on the round end scroll and in the crew report popup.
cap1

## Why's this needed? 
Some of these things used to be displayed on round end years ago, like 2013 and earlier, but they were removed for some reason. I added the ones I remembered from that time: richest escapee, most damaged escapee. And added a few others of my own. Just some fun stats to see for the end of the round really.

### CODE REVIEW

**Comment by pali6** [comment]
Note that when that code gets processed things might leave the shuttle and be in various other centcom areas instead (in which case they wouldn't count). But that's a more overaching problem tbh.

File: code/procs/scorestats.dm [hunk 1]
```diff
@@ -190,6 +200,93 @@ var/datum/score_tracker/score_tracker
...
+	proc/escapee_facts()
+		. = ""
+		//Richest Escapee | Most Damaged Escapee | Dr. Acula Blood Total | Clown Beatings
+		if (richest_escapee)		. += "<B>Richest Escapee:</B> [richest_escapee.real_name] : $[richest_total]<BR>"
+		if (most_damaged_escapee) 	. += "<B>Most Damaged Escapee:</B> [most_damaged_escapee.real_name] : [most_damaged_escapee.get_damage()]%<BR>"		//it'll be kinda different from when it's calculated, but whatever.
+		if (islist(pets_escaped))
+			var/who_escaped = ""
+			for (var/atom/A in pets_escaped)
+				who_escaped += " [A.name] [bicon(A)],"
+			. += "<B>Pets Escaped:</B>[copytext(who_escaped, 1, length(who_escaped)-1)]<BR><BR>"
```
Comment (ZeWaka) [inline code/procs/scorestats.dm:283]: What's with this weird copytext?

**Comment by ZeWaka** [comment]
Also: the old stats were removed by ISN during the round score rework that they never finished, if you care to know

**Comment by flrsh** [comment]
That Dr. Acula blood total is just straight up favoritism.

**Comment by pali6** [comment]
Are you suggesting we add Morty blood total too?

File: code/procs/scorestats.dm (hunk 1 above)
Comment (Kyle2143) [reply code/procs/scorestats.dm:283]: Uh, that's to remove the hanging comma at the end.

**Comment by Kyle2143** [comment]
Regarding pali's first comment. Yeah, that is an issue. During testing I had been searching all of centcom. Unfortunately, due to some mobs that exist in admin offices and whatnot, those would sometimes win the escapee stats. Which I didn't like.

File: code/procs/scorestats.dm (hunk 1 above)
Comment (pali6) [reply code/procs/scorestats.dm:283]: It's a minor optimization in the grand scheme of things but probably use `var/list/who_escaped` and then do `who_escaped.Join(",")`, not only will it be faster but you'll also avoid the copytext.

**Pull request merged**